Content.Server/Traitor/Uplink/UplinkSystem.cs

```diff
@@ -234,11 +31,20 @@ public bool AddUplink(EntityUid user, UplinkAccount account, EntityUid? uplinkEn
                     return false;
             }
 
-            var uplink = uplinkEntity.Value.EnsureComponent<UplinkComponent>();
-            SetAccount(uplink, account);
+            var store = EnsureComp<StoreComponent>(uplinkEntity.Value);
+            _store.InitializeFromPreset(uplinkPresetId, store);
+            store.AccountOwner = user;
+            store.Balance.Clear();
+
+            var currency = new Dictionary<string, FixedPoint2>
+            {
+                { "Telecrystal", balance }
```

Comment:
Isn't there a TcCurrencyPrototype defined somewhere?

Reply:
"Telecrystal" is the ID of that prototype. UplinkSystem doesn't have a component so I suppose i could just make it a const.